src/histolab/scorer.py

```diff
@@ -66,15 +67,59 @@ def __call__(self, tile: Tile) -> float:
         return np.random.random()
 
 
+class CellularityScorer(Scorer):
+    """Implement a basic Scorer that estimates the cellularity in an H&E-stained tile.
```

Comment:
why `basic`?

Reply:
Because it just uses the hematoxylin channel so it is not very specific... maybe general instead of basic?

Reply:
I don't see the need to use any specifier actually